meson.build

```
@@ -43,7 +43,7 @@ else
   os = option_os
 endif
 
-if os != 'Linux'
+if os == 'Dragonfly' or os == 'FreeBSD' or os == 'NetBSD' or os == 'OpenBSD'
```

instead of hardcoding almost all the BSD OSes, this can be simplified as:
```suggestion
if os.contains('BSD') or os == 'Dragonfly'
```